plugins/woocommerce/src/Admin/RemoteInboxNotifications/README.md

```diff
@@ -183,6 +188,22 @@ onboarding profile:
 }
 ```
 
+#### range
+
+`range` operator can check if a number falls within a certain range
```

Comment:
This would be a good place to note its `inclusive`

Reply:
@psealock Good suggestion! I'll make the change.

Reply:
Updated in https://github.com/woocommerce/woocommerce/pull/45201/commits/f9b3cf2ce9dcf15a80aa136a92f8f77601fa4c77